### ISSUE TITLE
Enabling pimatic-homeduino gives EACCES exception on startup.

### ISSUE THREAD

**Issue by Noxeus** [pimatic/pimatic-homeduino#62]
Yesterday everything worked fine, but now when I restart pimatic I get this error:

`14:30:08.466 [pimatic] info: New device "Zitkamer RF"...`
`14:30:09.866 [pimatic] error: A uncaught exception occured: Error: spawn EACCES`
`14:30:09.866 [pimatic] error:>    at errnoException (child_process.js:980:11)`
`14:30:09.866 [pimatic] error:>    at Process.ChildProcess._handle.onexit (child_process.js:771:34)`

I tried installing the old version `npm install pimatic-homeduino@0.8.78` but that didn't work sadly.
I know it's homeduino because when I disable the plugin pimatic loads fine.

**Comment by mwittig** [comment]
> but that didn't work sadly.

What did not work? Try to rempove the plugin directory before re-installing it, e.g.

```
cd pimatic-app/node_modules
sudo rm -rf pimatic-homeduino
sudo npm i --unsafe-perm pimatic-homeduino@0.8.78
sudo service pimatic restart
```

Please check the log then and let me known if the EACCES exception  still occurs. Can you also provide a full log, e.g. by adding it to a Gist, please?

**Comment by Kalekulan** [comment]
I got the same issue when I installed 0.8.80.

**Comment by mwittig** [comment]
@Kalekulan ok, can you pleease provide me with more info then? Thanks!

-  Do you use GPIO devices with pimatic-homeduino?
- Can you also provide a full log, e.g. by adding it to a Gist, please?

**Comment by Kalekulan** [comment]
Sorry for that. Yes, I use GPIOs with homeduino.

<script src="https://gist.github.com/Kalekulan/cd75734d049fed5ddc5894b61f264619.js"></script>

**Comment by mwittig** [comment]
No worries. I think I have found the problem. When the GPIO driver is used, it execute the vhduino binary. For some reason the binary has no execute bits set which causes the EACCESS error. I have added a post-install script to ensure the execute bits will beset and will publish a new release shortly.

**Comment by mwittig** [comment]
I think the issue should be fixed now with version 0.8.82. Please let me know if it solves the issue for you

**Comment by Kalekulan** [comment]
It works. Thanks!
